src/Api/ExportInterface.php

```diff
@@ -109,27 +110,27 @@ public function getReviewsRaw($limit = null, $offset = null, array $uids = []);
     /**
      * Returns JSON to comply with Mage 2 WebApi
      *
-     * @link http://developer.shopgate.com/plugin_api/export/get_reviews
-     *
      * @param string $action     - Requested method, e.g. get_items
      * @param string $shopNumber - Your shop number as configured in the merchant API
      * @param string $traceId    - Unique request trace ID, handshake related
      * @param string $limit      - limit number of reviews to pull
      * @param string $offset     - the offset number to pull from, e.g. 10 means next $limit from 10th item
-     * @param array  $uids       - array of review ID's to pull
+     * @param int[]  $uids       - array of review ID's to pull
+     *
+     * @link http://developer.shopgate.com/plugin_api/export/get_reviews
      *
      * @return string
      */
-    public function getReviews($action, $shopNumber, $traceId, $limit = null, $offset = null, array $uids = []);
```

Comment:
wait a moment, not all array things were removed (e.g. ```getItems()```) from the interface and the class ```cart-integration-magento2-base/src/Model/Service/Export.php``` still references the old ```array $uids = []``` for getReviews(). Same goes for the Export module reference, it still references the old.

Reply:
These are not validated. Debugging the validation in getParamTypes shows that only getCategories is checked for the PHPdoc.
If they are on your instance, please let me know. Otherwise, I would not want to touch them

Reply:
It's good now, with your export fix :)
Created a few tests for postman as well as automatic pre-authentication for framework calls, no need to modify apis.php anymore with ```defined('SHOPGATE_DEBUG')```. Will post details in the ticket.